apps/messages/widget.js

```diff
@@ -51,5 +53,6 @@ WIDGETS["messages"]={area:"tl", width:0, draw:function(recall) {
 /* We might have returned here if we were in the Messages app for a
 message but then the watch was never viewed. */
 if (global.MESSAGES===undefined)
-  WIDGETS["messages"].update(require("messages").getMessages());
+  WIDGETS["messages"].onMsg('',{});
+  Bangle.on("message", WIDGETS["messages"].onMsg);
```

Comment:
If MESSAGES is defined and we're in the app, don't we still want to register our `Bangle.on("message"` handler?

Reply:
I think I'm not really sure what's going on here actually...
Do we even want to load the widget while in the messages app?

Reply:
That's a good question, maybe not... I just pushed a change to the messages lib that makes it 'fast load' the app if it can (which is really neat because it makes it much more responsive). But, that means that if the messages widget was already loaded, it'll stay loaded - so would have to cope with being around for the messages app anyway.